extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/CodeAuthenticationMechanism.java

```diff
@@ -497,7 +542,8 @@ private boolean isForceHttps(TenantConfigContext configContext) {
     private AuthenticationRedirectException redirectToLogoutEndpoint(RoutingContext context, TenantConfigContext configContext,
             String idToken) {
         removeCookie(context, configContext, getSessionCookieName(configContext));
-        return new AuthenticationRedirectException(buildLogoutRedirectUri(configContext, idToken, context));
+        return new AuthenticationRedirectException(shouldAutoRedirect(configContext, context),
+                buildLogoutRedirectUri(configContext, idToken, context));
```

Comment:
Same here. This constructor doesn't exist for AuthenticationRedirectException. The original code at 500 is the correct version.